mandatory/2-function-creation.js

```diff
@@ -25,7 +32,10 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
```

Comment:
🐞 Tiny typo here that is causing the automated test to fail.
Hint : Check your '"' in one of the string methods ( they come in pairs, one to "start" the string and one to "end" the string)

Reply:
Still giving me an error on line 13 with the ".trim()".

   34 | */
      35 | function tidyUpStrings(arrayOfStrings) {
    > 36 |   let strings = arrayOfStrings.trim().replace("/", "").toLowerCase();
         |                                                 ^
      37 |   return strings;
      38 | }
      39 |

Reply:
trim() is a function that works on a string, but the argument being passed into the function ( parameter arrayOfStrings) is an array of string.
We covered array methods recently; what array method can we use to perform a function on each of the array elements and return a new array?